resources/views/fixometer/device-add-or-edit.blade.php

```diff
@@ -0,0 +1,231 @@
+<form class="add-device" data-device="{{ $device->iddevices }}" method="post" enctype="multipart/form-data">
+    <div class="collapse" id="add-edit-device-{{ $powered ? 'powered' : 'unpowered' }}-{{ $device->iddevices }}">
+        <div class="device-info">
+            <div class="card-event-add-item card flex-grow-1 border border-top-0 border-bottom-1 border-left-0 border-right border-white">
+                <div class="card-body d-flex flex-column">
+                    <h3>ITEM</h3>
+                    <div class="mt-4">
+                        <div class="mb-2 device-select-row">
+                            <div class="form-control form-control__select form-control-lg d-inline">
+                                <select name="category" class="category select2">
+                                    <option value="">@lang('devices.category')</option>
+                                    @foreach( $clusters as $cluster )
+                                        @php
+                                        $empty = true;
+
+                                        foreach( $cluster->categories as $category ) {
+                                            if ($powered && $category->powered || !$powered && !$category->powered) {
+                                                $empty = false;
+
+                                            }
+                                        }
+                                        @endphp
+                                        @if (!$empty)
+                                            <optgroup label="{{{ $cluster->name }}}">
+                                                @foreach( $cluster->categories as $category )
+                                                    @if ($powered && $category->powered || !$powered && !$category->powered)
+                                                        @if( $device->category == $category->idcategories)
+                                                            <option value="{{{ $category->idcategories }}}" selected>{{{ $category->name }}}</option>
+                                                        @else
+                                                            <option value="{{{ $category->idcategories }}}">{{{ $category->name }}}</option>
+                                                        @endif
+                                                    @endif
+                                                @endforeach
+                                            </optgroup>
+                                        @endif
+                                    @endforeach
+                                    @if( $device->category == 46 )
```

Comment:
One day we should remove this magic constant for Misc (powered) that we use everywhere - but let's not take that on just now!